khal/icalendar.py

```diff
@@ -22,10 +22,13 @@
 """collection of icalendar helper functions"""
 
 import datetime as dt
+try:
+    from zoneinfo import ZoneInfo
+except ImportError:
+    from backports import zoneinfo as ZoneInfo
```

Comment:
For some reason flake8 does not like this. Any ideas on how to fix it?

Reply:
is this problem still up? Is there a full log of the error. I will look into it and come back